airgun/entities/host.py

```diff
@@ -419,6 +419,15 @@ def get_displayed_table_headers(self):
         view.wait_displayed()
         return view.displayed_table_header_names
 
+    def build_mode(self, entity_name, value=None):
```

Comment:
`Added a method to verify the facts for the host.` Can you please more elaborate on the PR description and add some docstring to your entity and maybe alter the entity name so it more reflects what the entity is doing. Thanks!